df.entities.xml

```diff
@@ -143,41 +143,41 @@
         </bitfield>
     </struct-type>
 
-    <struct-type type-name='caravan_state'>
-        <int32_t name='total_capacity'/>
-        <int32_t name='unk_1'/>
-        <enum name='trade_state' base-type='int8_t'>
+    <struct-type type-name='caravan_state' original-name='plot_merchantst'>
+        <int32_t name='total_capacity' comment='bay12: supportedweight; actually a massst'/>
+        <int32_t name='total_capacity_fraction' comment='fraction part of total_capacity'/>
+        <enum name='trade_state' base-type='int8_t' comment='bay12: stage'>
             <enum-item name='None'/>
             <enum-item name='Approaching'/>
             <enum-item name='AtDepot'/>
             <enum-item name='Leaving'/>
             <enum-item name='Stuck'/>
         </enum>
-        <int8_t name='depot_notified' comment='has it warned you that you need a depot'/>
-        <int16_t name='time_remaining'/>
+        <int8_t name='depot_notified' comment='bay12: havecomplained; has it warned you that you need a depot'/>
+        <int16_t name='time_remaining' comment='bay12: timer'/>
         <int32_t name='entity' ref-target='historical_entity'/>
-        <compound name='activity_stats' type-name='entity_activity_statistics'/>
+        <compound name='activity_stats' type-name='entity_activity_statistics' comment='bay12: report'/> reportst
         <bitfield name='flags' base-type='uint32_t'>
-            <flag-bit name='check_cleanup' comment='set each time a merchant leaves the map or dies'/>
-            <flag-bit name='casualty'/>
-            <flag-bit name='hardship'/>
-            <flag-bit name='communicate' comment='send data to mountainhomes'/>
-            <flag-bit name='seized'/>
-            <flag-bit name='offended'/>
-            <flag-bit name='announce' comment='display merchantintro and merchantexit'/>
-            <flag-bit name='greatly_offended'/>
-            <flag-bit name='tribute' comment='caravan is delivering tribute (not merchant caravan)'/>
+            <flag-bit name='check_cleanup' comment='CHECKANDFINALIZE; set each time a merchant leaves the map or dies'/>
+            <flag-bit name='casualty' comment='JUSTSPOILED'/>
+            <flag-bit name='hardship' comment='WENTBADLY'/>
+            <flag-bit name='communicate' comment='SOMEBODYLIVED; send data to mountainhomes'/>
+            <flag-bit name='seized' comment='GOODSSEIZED'/>
+            <flag-bit name='offended' comment='NOMORETRADE'/>
+            <flag-bit name='announce' comment='UNUSED_07 (used to be FIRST_MERCHANT) display merchantintro and merchantexit (???)'/>
```

Comment:
In older versions, `data/announcement/merchantintro`:
```
merchantintro
[TITLE]The Autumn Caravan[/TITLE]
[P]Although it is a grim reminder of the winter hardships to come,
the supply caravan from [VAR:NATIVENAME:GLOBAL:YOURCIV] is a welcome sight.
As the merchants unload their goods at the depot,
you notice with pride that their eyes are alight with
the anticipation of inspecting
the splendid products of your industrious craftsdwarves.
Take careful stock of your own stores.  What these dwarves offer
might very well be the difference between a prosperous future and
a slow and meaningless death.
```
and `data/announcement/merchantexit`:
```
merchantexit
[TITLE]The Caravan's Departure[/TITLE]
[P]The dwarves begin packing their belongings.
```
...which used to be much longer back in 0.23
```
merchantexit
[TITLE]The Caravan's Departure[/TITLE]
[P]The dwarves begin packing their belongings.  They advise you
that wide roads, and bridges if necessary, are the surest way
to attract the well-stocked wagons of human traders that
travel out from their vast cities above the earth.
Be sure that the road leads right to your depot.  These humans
aren't very hardy; rough terrain taxes them.  Even raw stone floors!  Bah!
```